client/js/apiKeys.js

```diff
@@ -0,0 +1,7 @@
+const getApiKeyFromStorage = async () => {
```

Comment:
Why is API being fetched from local cache? I mean, API shouldn't be on session/browser storage. We need to come up with something for this.

Reply:
API is not fetched from local storage. For API we need key which is stored in chrome sync storage. This is different than local storage as its synced across browser sessions.

For now, our API key is being used but in future we can present a way for user to give their own keys so that they can be stored for use. That's why we have this code.